lighthouse-core/gather/gatherers/dobetterweb/tags-blocking-first-paint.js

```diff
@@ -147,15 +147,14 @@ class TagsBlockingFirstPaint extends Gatherer {
     return options.driver.evaluateScriptOnLoad(scriptSrc);
   }
 
+  /**
+   * @param {!Object} options
+   * @param {{networkRecords: !Array<!NetworkRecord>}} tracingData
```

Comment:
record types like this are non-nullable, so the `!` is technically correct but unnecessary. It would have been really nice if everything had been non-nullable by default from the start :(